maintenance/wikia/createUser.php

```diff
@@ -35,7 +35,7 @@
 if ( !is_object( $user ) ) {
 	echo( "invalid username.\n" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the parentheses from this "echo" call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2041)

[project-id]: wikia:app